src/Mai/mappings/handlers.ts

```diff
@@ -0,0 +1,85 @@
+import { Address, BigInt, Bytes, log } from "@graphprotocol/graph-ts";
+import { MaiCamTokenData } from "../../../generated/schema";
+import { MaiAToken } from "../../../generated/MaiCamAavecamAAVE/MaiAToken";
+import { MaiLendingPool } from "../../../generated/MaiCamAavecamAAVE/MaiLendingPool";
+import { convertBINumToDesiredDecimals, toAddress } from "../../utils/converters";
+
+import { MaiIncentivesController } from "../../../generated/MaiCamTokencamUSDC/MaiIncentivesController";
+import { ZERO_BD, ZERO_BI } from "../../utils/constants";
+
+let CAM_INCENTIVES_CONTROLLER_ADDRESS: Address = toAddress("0x357D51124f59836DeD84c8a1730D72B749d8BC23");
```

Comment:
This can go into the constants file.

Reply:
Done. On the other hand, I let the DECIMALS constant inside the scope of the `convertBINumToDesiredDecimals()` function that was using it.